app/src/main/res/values/strings.xml

```diff
@@ -3,9 +3,9 @@
 
     <string name="title_section1">Discover</string>
     <string name="title_section2">Make</string>
-    <string name="title_activity_map">Map</string>
     <string name="title_activity_project">Project</string>
-    <string name="title_activity_editor">Options</string>
+    <string name="title_activity_page">Page</string>
+    <string name="title_activity_element">Options</string>
```

Comment:
I agree. This matches the spec more closely, but something to consider. I recommend using what Kate has here but filing a follow-up ticket.